game-core/src/main/java/games/strategy/engine/data/ResourceList.java

```diff
@@ -5,6 +5,9 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * A keyed collection of {@link Resource}s.
+ */
```

Comment:
should we get a TODO comment to rename this? It seems too comfortably out of sync that this is called 'ResourceList' and the comment and implementation are a keyed collection.

Reply:
Ditto above comment.  Can't rename in order to maintain backwards compatibility.